tests/baselines/reference/showConfig/Shows tsconfig for single option/pretty/tsconfig.json

```diff
@@ -1,3 +1,5 @@
 {
-    "compilerOptions": {}
+    "compilerOptions": {
+        "pretty": true
```

Comment:
Do we need to show output formatting options ?

Reply:
`locale` doesn't work when set in a `tsconfig.json` - I tested it for the TSConfig reference last year, that _could_ be considered a bug maybe, but at least this does reflect reality.

`pretty` in the tsconfig on the other hand _does_ affect output:

```sh
❯ yarn tsc
index.ts(3,1): error TS2588: Cannot assign to 'a' because it is a constant.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

# Edit tsconfig

❯ yarn tsc
index.ts:3:1 - error TS2588: Cannot assign to 'a' because it is a constant.

3 a= b
  ~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

 ~/dev/typescript/repros/pretty ❯
```

Which means this is _accurate_ to my knowledge

Reply:
Yes this is accurate but what i am asking is do we want to show these options when --showConfig is run?

Reply:
No, will fix this 👍🏻